### PR TITLE
ceph-docker: run flake8 and lint jobs on 2.jenkins

### PR DESCRIPTION
**Pull request by djgalloway**
Signed-off-by: David Galloway <dgallowa@redhat.com>

### CODE REVIEW

File: ceph-docker-flake8/build/build [hunk 1]
```diff
@@ -29,7 +29,7 @@ function check(){
 
 function main() {
     # install some of our dependencies
-    if [ "${HUDSON_URL}" = "https://jenkins.ceph.com/" ]
+    if [ "${HUDSON_URL}" = "https://2.jenkins.ceph.com/" ]
```
Comment (andrewschoen) [inline ceph-docker-flake8/build/build]: Why this condition at all? Was somebody trying to build this job so it could be run locally?

Comment (djgalloway) [reply ceph-docker-flake8/build/build]: Looks like: https://github.com/ceph/ceph-build/commit/83b99762805edd3bf133ee1a2d02674b90c241c5

Comment (djgalloway) [reply ceph-docker-flake8/build/build]: Maybe change it to check if the env var is set at all?

Comment (andrewschoen) [reply ceph-docker-flake8/build/build]: Yeah, I think that makes better sense. Maintaining jenkins urls in jobs will get tedious if we need to add more master nodes.

**Comment by djgalloway** [comment]
@andrewschoen updated to just check if var is set.

**Pull request merged**